solr/core/src/test/org/apache/solr/cloud/ReindexCollectionTest.java

```diff
@@ -167,24 +167,23 @@ public void testSameTargetReindexing() throws Exception {
 
   private void doTestSameTargetReindexing(boolean sourceRemove, boolean followAliases)
       throws Exception {
-    final String sourceCollection = "sameTargetReindexing_" + sourceRemove + "_" + followAliases;
-    final String targetCollection = sourceCollection;
```

Comment:
source vs. target collection naming seems helpful here w.r.t. test logic readability

Reply:
agreed...   changing this back...  Thanks!